CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## 1.10.0-beta.1
+
+### ✨ Features
+
+* Add `mapboxgl.preloadWorkerPool()` and `mapboxgl.removePreloadedWorkerPool()` methods to allow developers to optimize load times for their maps ([#9391](https://github.com/mapbox/mapbox-gl-js/pull/9391))
+* Add `index-of` and `slice` expressions to search arrays and strings for the first occurrence of a specified value and return a section of the original array or string ([#9450](https://github.com/mapbox/mapbox-gl-js/pull/9450)) (h/t [lbutler](https://github.com/lbutler))
+* Correctly set RTL text plugin status if the plugin URL could not be loaded. This allows developers to add retry logic on network errors when loading the plugin ([#9489](https://github.com/mapbox/mapbox-gl-js/pull/9489))
+
+### Gestures ([#9365](https://github.com/mapbox/mapbox-gl-js/pull/9365))
+
+- You can now interact with multiple maps on the same page at the same page. For example, pinch rotate one map while touch panning a different one
+add a second finger, keep panning, remove the first, keep panning
```

Comment:
🤔 I'm quite confused by this example: `For example, pinch rotate one map while touch panning a different one
add a second finger, keep panning, remove the first, keep panning` -- was something accidentally deleted from L12?

In any case, I think "You can now interact with multiple maps on the same page at the same page." is sufficiently clear without needing an example -- people can head to the PR for details,  so I recommend removing.